app/src/atoms/buttons/SmallButton.tsx

```diff
@@ -34,6 +34,8 @@ interface SmallButtonProps extends StyleProps {
   iconName?: IconName | null
   buttonCategory?: ButtonCategory // if not specified, it will be 'default'
   disabled?: boolean
+  /** aria-disabled for displaying snack bar */
+  ariaDisabled?: boolean
```

Comment:
can you make the comment a bit more descriptive? something like this: 


```suggestion
  /** aria-disabled for displaying snack bar, used for ODD only at this time. */
  ariaDisabled?: boolean
```

it would be good to mention that this is only for ODD components at the moment, which i know `Smallbutton` is only used for ODD but it is still good to give additional details.